### ISSUE TITLE
I cannot reproduce CAER-S datasets

### ISSUE THREAD

**Issue by ByeongjunCho** [Talented-Q/POSTER_V2#15]
Hi. Thanks to share code

I reproduced with CAER-S datasets but I got poor result

![image](https://github.com/Talented-Q/POSTER_V2/assets/34848770/80ec1d13-9162-4dd8-a850-2cbeb74ea9b8)

I downloaded CAER-S as below
https://caer-dataset.github.io

And CAER-S data structure as below
 ``` 
data/CAER-S/
    CAER-S/
            	   train/
                            Angry/
                                        0001.png
                                        0002.png
 	                                ...
 	           valid/
 	                    Happy/
                                        0001.png
                                        0002.png
                            Sad/
                                        0001.png
                                        0002.png
 	                               ...
                                       ```

And, have some problems
Q1. dataset need preprocessing? example) alignment, ...
Q2. If datasets need preprocessing, Can you share processing code?

Thank you!

Byeongjun Cho

**Comment by Talented-Q** [comment]
Thank you for your attention, we did not pre-process CAER-S and directly used the CAER-S dataset. By looking at the confusion matrix generated by the code in your image, each type of expression contains more than 90% of the cases identified as an expression (e.g. 2850 in the first line, 2951 in the second line, 2982 in the third line....). We doubt whether the label order of expressions does not correspond well. Please check the expression label of the training set and the test set in the dataset.

**Comment by ByeongjunCho** [comment]
Oh. It looks label order is wrong. Thank you. I close this issue. 

Thank you.
